src/pages/content-edit.js

```diff
@@ -11,6 +11,8 @@ var m      = require("mithril"),
     update   = require("../lib/update"),
     watch    = require("../lib/watch"),
 
+    capitalize = require("lodash.capitalize"),
```

Comment:
Gonna be a butt and ask you to move this up w/ all the other `node_modules` declarations.